app/controllers/clients_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ClientsController < ApplicationController
+
+  # GET /
+  def index
+  end
+
+  # GET /client
+  def show
+  end
+
+  def create
+    client = Client.new client_params
+    client.save!
+  end
+
+  # GET /client/edit
+  def edit
```

Comment:
I dont think we will need the edit action for an api.  We only need to be able to update on the back end